spec/cddl/types.cddl

```diff
@@ -31,7 +31,14 @@ range<T> = {
     ? 1 => bound<T>,
 }
 
-; An attribute related index, which can include sub-attributes or IDs recursively.
 attribute-related-index     = attribute-id / sub-attribute-related-index
+
+; An attribute related index, which can include sub-attributes or IDs recursively.
+std-attribute-related-index = std-attribute-id / std-sub-attribute-related-index
```

Comment:
`std-sub-attribute-related-index` is not defined. Did you mean `sub-attribute-related-index`?